README.md

```diff
@@ -59,7 +65,7 @@ If you want to customize the look and feel of your admin, you should use the SAS
 
 ```css
 @import "admin_variables";
-@import "../../../node_modules/torch/assets/css/torch";
+@import "~torch/assets/css/index";
```

Comment:
Is there a particular reason you changed the name of the css file from `torch` to `index` ?